BossMod/Autorotation/InputOverride.cs

```diff
@@ -30,6 +30,9 @@ class InputOverride : IDisposable
         private delegate ref int GetRefValueDelegate(int vkCode);
         private GetRefValueDelegate _getKeyRef;
 
+        private IntPtr _forceDisableMovementPtr;
+        internal unsafe ref int ForceDisableMovement => ref *(int*)(_forceDisableMovementPtr + 4);
```

Comment:
Interesting, this seems to be some sort of an internal counter in PlayerController that's used to lock player during animations...